packages/wonder-blocks-dropdown/src/components/multi-select.tsx

```diff
@@ -569,6 +569,7 @@ const MultiSelect = (props: Props) => {
                             disabled={isDisabled}
                             id={uniqueOpenerId}
                             aria-controls={dropdownId}
+                            placeholder={noneSelected}
```

Comment:
Should the placeholder be `menuText` so the aria-label changes when the selected values change? Otherwise, it'll always have an aria-label for no items selected. `menuText` isn't always a string though since it can be custom jsx! Or maybe the select opener wouldn't need an aria-label since the component should be used with a `<label>`? 🤔 What do you think!

Here's a  screenshot that shows the aria-label with "0 items" while the component shows "3 items":
<img width="1440" alt="Screenshot 2025-01-09 at 4 05 35 PM" src="https://github.com/user-attachments/assets/06a4176d-de34-4c56-8aed-5e58842d2d29" />

Reply:
Good catch @beaesguerra! I don't think that `aria-label` is necessarily the same as a placeholder for a combobox. This text about the number of items is more of a value than a label. I'm guessing the intent was to use a placeholder as a label in the absence of a separate label element (or associated label, however it's marked up). But we have to be careful with `aria-label` to not override an associated label, if it's paired with `<label for="$idMatchingThisCombobox">`. I'm looking into it!

Reply:
@jandrade or @beaesguerra do you know if the MultiSelect component is used with an explicit label (`<label for="thisSelectComponentWithMatchingID">`? I don't see a story for it like there is on [SingleSelect](https://khan.github.io/wonder-blocks/?path=/docs/packages-dropdown-singleselect-accessibility--docs).

I'll dig into this more -- but I'm wondering if we have any way to conditionally add an `aria-label` to the opener for placeholders and not clash with other labeling mechanisms. Any input you can add would be helpful!

Reply:
It should be used with a label, specifically LabeledField 😄 Here's an example in Storybook: https://5e1bf4b385e3fb0020b7073c-fnsnwmxvwb.chromatic.com/?path=/story/packages-labeledfield--fields

Reply:
Agreed, we should use `LabeledField` (or `label` on very specific cases), but there might be places where we are not using labels and `aria-label` could be useful. For example, the course selector in the teacher dashboard.

You'll need to have "Teacher" access (available with your @khanacademy.org account), then go to `/teacher/dashboard`, click on any class and use the dropdown on the top left.

<img width="1300" alt="Screenshot 2025-01-14 at 2 59 58 PM" src="https://github.com/user-attachments/assets/abf3326a-1f7e-4943-b54f-1f6c77a51383" />

NOTE: This could be a case where we could benefit from having an actual label and this could be validated with the TX team.

Reply:
Oh yes, `LabeledField` 🤦‍♀️! Thank you so much for this context. I think the challenge is at the Combobox level, where it won't know whether it's being used by labeled field. Fortunately I think we can leverage the Accessible Name Computation spec to prioritize the LabeledField label over an `aria-label` (in the event both are present). We can also instruct consumers when and when not to use `aria-label` in docs!